packages/mui-system/src/createStyled.test.js

```diff
@@ -240,14 +240,23 @@ describe('createStyled', () => {
           },
         },
       });
-      const { container } = render(
-        <ThemeProvider theme={finalTheme}>
-          <Button>Hello</Button>
-        </ThemeProvider>,
-      );
-      expect(container.firstChild).toHaveComputedStyle({
-        width: '300px',
-      });
+      try {
+        // Not sure why it throws another error.
+        // If you know, please submit a PR.
+        expect(() => {
+          render(
+            <ThemeProvider theme={finalTheme}>
+              <Button>Hello</Button>
+            </ThemeProvider>,
+          );
+        }).toErrorDev(
+          `MUI: The return type of the callback in 'MuiButton.styleOverrides.root' cannot be a string. Change the return type to array or object instead.`,
+        );
+      } catch (error) {
+        expect(error.message).to.equal(
+          `MUI: The return type of the callback in 'MuiButton.styleOverrides.root' cannot be a string. Change the return type to array or object instead.`,
+        );
+      }
```

Comment:
@mnajdova test added but I could not figure out why it throws another error, do you know?